src/routes/users.routes.ts

```diff
@@ -7,6 +7,10 @@ export default function usersRoutes(api: Router) {
     .route('/users')
     .get(requireRolePermission('users', 'read'), UsersCtrl.list);
 
+  api
+    .route('/users/me')
```

Comment:
Do you even need this endpoint? Shouldn't you just be searching for users by id only?

Reply:
meh, i like having it, i've used it in various apps in the past and figured i'd add the functionality.

Reply:
APPROVED!